--- splix-1.1.1/Makefile.optflags	2008-09-20 21:20:29.000000000 +0200
+++ splix-1.1.1/Makefile	2008-01-27 23:52:00.000000000 +0100
@@ -4,8 +4,8 @@
 #  This project has been placed under the GPL Licence.
 #
 
-CXXFLAGS	:= -O2 `cups-config --cflags` 
-LDFLAGS		:= `cups-config --ldflags`
+CXXFLAGS	:= $(OPTCXXFLAGS) `cups-config --cflags` 
+LDFLAGS		:= $(OPTLDFLAGS) `cups-config --ldflags`
 CUPSFILTER	:= `cups-config --serverbin`/filter
 CUPSPPD		:= `cups-config --datadir`/model
 
--- splix-1.1.1/src/Makefile~	2008-01-27 23:52:00.000000000 +0100
+++ splix-1.1.1/src/Makefile	2008-09-20 21:24:42.000000000 +0200
@@ -4,7 +4,7 @@
 #  This project has been placed under the GPL Licence.
 #
 
-CXXFLAGS	+= -I../include -Wall -g -O0 #-DENABLE_DEBUG
+CXXFLAGS	+= -I../include -Wall #-DENABLE_DEBUG
 LDFLAGS		+=
 LIBS		+= -lcups -lcupsimage
 
